### PR TITLE
HOC Wrapper - Loading

### PR DESCRIPTION
**Pull request by sfernandez11**
# Summary
## Features
* Added a HOC Wrapper when LogIn and get the user's profile are loading.
* When the Component is loading, a spinner show up in the screen.
## Screenshots
image
## Trello Card
https://trello.com/c/PZZ7EQRI/20-patterns-ii-hocs

### CODE REVIEW

File: src/app/screens/LogIn/index.js [hunk 1]
```diff
@@ -15,15 +18,19 @@ class LogIn extends Component {
...
-        <h2 className="logIn-form-title">{strings.ticTacToeTitle}</h2>
-        <LogInForm onSubmit={this.handleSubmit} />
+        <LogInFormWithLoading isLoading={this.props.loading} onSubmit={this.handleSubmit} />
       </div>
     );
   }
 }
 
+LogIn.propTypes = {
+  loading: PropTypes.bool
```
Comment (IgnacioAbadie) [inline src/app/screens/LogIn/index.js]: required

File: src/app/screens/Profile/index.js [hunk 2]
```diff
@@ -27,12 +29,14 @@ class Profile extends Component {
 
 Profile.propTypes = {
   userProfile: PropTypes.node,
-  error: PropTypes.string
+  error: PropTypes.string,
+  loading: PropTypes.bool
```
Comment (IgnacioAbadie) [inline src/app/screens/Profile/index.js]: required

**Review by IgnacioAbadie — APPROVED** [review]

File: src/app/screens/LogIn/index.js (hunk 1 above)
Comment (sfernandez11) [reply src/app/screens/LogIn/index.js]: Done

File: src/app/screens/Profile/index.js (hunk 2 above)
Comment (sfernandez11) [reply src/app/screens/Profile/index.js]: Done

**Pull request merged**